src/styles/product-card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 0;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    height: 134px;
+    width: 160px;
+    margin: 0 3px 40px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__code {
+    padding-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $add-text-color;
+  }
+
+  &__revs {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 16px;
+    margin-bottom: 24px;
+  }
+
+  &__revs-text {
+    font-size: 10px;
+    line-height: 16px;
+    font-weight: 400;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    font-size: 12px;
+    line-height: 18px;
+    color: $add-text-color;
```

Comment:
The variable `$add-text-color` is used here again. Ensure it is defined in your SCSS files to avoid compilation errors.